frontend/appflowy_flutter/lib/user/presentation/screens/sign_in_screen/mobile_sign_in_screen.dart

```diff
@@ -24,7 +24,7 @@ class MobileSignInScreen extends StatelessWidget {
                 mainAxisAlignment: MainAxisAlignment.center,
                 children: [
                   Text('Signing in...'),
```

Comment:
Localization

Reply:
It is a temporary loading effect there, I plan to remove/update it in the future(before releasing the app), so I didn't add localization.

Reply:
Then we can merge 🙏👌